packages/account/src/Sections/Security/AccountLimits/account-limits-info.tsx

```diff
@@ -17,7 +18,12 @@ const currency_name_map = {
     GBP: { display_code: 'GBP', name: localize('Pound Sterling') },
 };
 
-const AccountLimitsInfo = ({ currency, is_virtual }) => (
+type TAccountLimitsInfo = {
+    currency?: string;
+    is_virtual?: boolean;
+};
+
+const AccountLimitsInfo = ({ currency, is_virtual }: TAccountLimitsInfo) => (
```

Comment:
```suggestion
const AccountLimitsInfo = ({ currency, is_virtual = false }: TAccountLimitsInfo) => (
```
@yauheni-deriv It's better if we always give boolean props default value so the type will be `boolean` and not ` boolean | undefined` 🙇🏻